nustyle.css

```diff
@@ -376,3 +384,204 @@ ul.icons2 {
   align-self: center;
   margin-top: 15px;
 }
+
+#form-button:hover {
+  background-color: #f8b791;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#footer-hr {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #nav-mobile {
+    display: none;
+  }
+
+  #nav-desktop {
```

Comment:
- [Optional] This is my general view of your project @Nuga0909, I have attached a video, I hope it helps 👍😊


https://user-images.githubusercontent.com/69077061/199807953-04938490-cb91-4727-8866-ec3689fc000b.mp4